freqtrade/exchange/exchange.py

```diff
@@ -1244,7 +1307,7 @@ def get_fee(self, symbol: str, type: str = '', side: str = '', amount: float = 1
             # validate that markets are loaded before trying to get fee
             if self._api.markets is None or len(self._api.markets) == 0:
                 self._api.load_markets()
-
+            # TODO-lev: Convert this amount to contract size?
```

Comment:
Don't bother (for two reasons, actually).
The way we are using `calculate_fee` is to only take the rate (relative value - like 0.01%).

While amount is a necessary input for ccxt - it has no impact on the relative result - (0.01% will remain identical, for both 1 or for 100_000).
Freqtrade is also never passing in amount - so it'll always use the default of 1 anyway.